addons/website_event/static/src/js/website_event_set_customize_options.js

```diff
@@ -109,6 +109,13 @@ CustomizeMenu.include({
             if (!self.__eventOptionsLoaded && self._getEventObject().model === 'event.event') {
                 self.__eventOptionsLoaded = true;
                 self.eventOptions = new EventSpecificOptions(self);
+                // If this is the first customize menu, add the divider at top
+                if (!self.$('.dropdown-divider').length) {
```

Comment:
Looking at website code seems to be inlined with what is done in website CustomizeMenu. Pinging @qsm-odoo to be sure it is the right way to do it :) .

Reply:
I'd say the right way to do it is to refactor the website code so that there is a method to load/build the options and a method which adds them in the DOM. That second one would then be in charge of adding the divider if anything was loaded. Then website_event only has to `include` the loading/building part and the code which adds the dropdown-divider if necessary would not be duplicated :+1:

Reply:
Refactoring code is for master, not for stable :) . I prefer to keep diff minimal as this is a bugfix.

Reply:
Did not pay attention this was for 14.0, my bad :) Still ideas for the forward-ported version :point_up: :+1: